tasks/podman.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Install podman package.
+  package:
+    name: podman
+    state: present
+
+- name: Check homedir of retrace user.
+  shell: grep retrace /etc/passwd | cut -d ':' -f6
+  register: retrace_home
+
+- name: Set home directory for retrace user.
+  command: usermod retrace -d /var/lib/retrace
+  when: retrace_home.stdout != "/var/lib/retrace"
+
+- name: Check if subuid is set for retrace user.
+  shell: grep retrace /etc/subuid | cat
+  register: retrace_subuid
+
+- block:
+  - name: Get last subuid entry.
+    shell: tail -1 /etc/subuid | cut -d ':' -f2
```

Comment:
This assumes that the entries are neatly ordered. What about `sort`ing by that particular field?

Reply:
`sorted`